pkg/config/install_requirements_test.go

```diff
@@ -407,4 +407,7 @@ func Test_LoadRequirementsConfig(t *testing.T) {
 			}
 		})
 	}
+
+	_, _, err := config.LoadRequirementsConfig(path.Join(testDataDir, "jx-requirements-syntax-error"))
```

Comment:
Let's put this test into its own test function and give it a name which tells what we test, something like `Test_LoadRequirementsConfig_load_invalid_yaml`

I also think we should not only assert that an error is returned, but also that is the error we expect. For example by looking at the error message.